games/rtype/include/Components.hpp

```diff
@@ -0,0 +1,212 @@
+/*
+** EPITECH PROJECT, 2023
+** RType
+** File description:
+** Components.hpp
+*/
+
+#ifndef RTYPE_COMPONENTS_HPP
+#define RTYPE_COMPONENTS_HPP
+
+#include "OtterGraphic.hpp"
+
+#include <memory>
+#include <string>
+
+namespace Otter::Games::RType::Components {
+
+    /**
+     * @brief Component for the rendering
+     * @details The render component is used to render an entity
+     * @struct Render
+     */
+    struct Render {

```

Comment:
need to store data, it not the system who store data
extract sprite from encapsulation to put it in a composant, or put the encapsulation in the component render